src/graphql/root/mutation/user-login-device.ts

```diff
@@ -0,0 +1,58 @@
+import { GT } from "@graphql/index"
+
+import { Auth } from "@app"
+import { mapAndParseErrorForGqlResponse } from "@graphql/error-map"
+import JwtPayload from "@graphql/types/payload/jwt"
+import { BTC_NETWORK } from "@config"
+
+const UserLoginDeviceInput = GT.Input({
+  name: "UserLoginDeviceInput",
+  fields: () => ({
+    jwt: {
+      type: GT.String,
+    },
+  }),
+})
+
+const UserLoginDeviceMutation = GT.Field<{
+  input: {
+    jwt: string | InputValidationError
+  }
+}>({
+  extensions: {
+    complexity: 120,
+  },
+  type: GT.NonNull(JwtPayload),
+  args: {
+    input: { type: GT.NonNull(UserLoginDeviceInput) },
+  },
+  resolve: async (_, args, { ip }) => {
+    const { jwt } = args.input
```

Comment:
it's not great to have to pass the jwt around (as opposed to passing a checked value from the header from Oathkeeper mutator, because Oathkeeper would have checked the value already) but I guess this is how this new design architecture works

we should make sure the jwt is excluded from open telemetry

Reply:
The idea behind this was to make it similar to the phone login flow. The jwt is equivalent to the twilio code. The userLogin and userLoginDevice mutations are UnAuth'd called ( i.e no oathkeeper bearer).  So the userLoginDevice call is verfied in the `jwtVerfy` function similar to how the twilio code is validated with `isCodeValid`.

All `Auth'd` mutations and queries still get the jwt validated in oathkeeper and the subject is  passed to the backend for authenicated calls.

Reply:
note: this is an indication we should probably have a normal rest endpoint for this, and other typical authentication endpoint. ok, you can resolve back the conversation :)

Reply:
redacted jwt here in telemetry https://github.com/GaloyMoney/charts/pull/3158/commits/53a98af477acd9d9566ba42c9f856d729c0945ab